### PR TITLE
Add Hydra::Catalog as an analog to Blacklight::Catalog

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

**Comment by awead** [comment]
I'll test this against Sufia to make sure before merging.

**Comment by mjgiarlo** [comment]
Thanks, @awead

File: hydra-core/lib/generators/hydra/templates/catalog_controller.rb [hunk 1]
```diff
@@ -3,9 +3,7 @@
 
 class CatalogController < ApplicationController
 
-  include Blacklight::Catalog
-  include Hydra::Controller::ControllerBehavior
-  include Hydra::Controller::SearchBuilder
+  include Hydra::Catalog
```
Comment (awead) [inline hydra-core/lib/generators/hydra/templates/catalog_controller.rb]: I don't see where Hydra::Catalog also includes Hydra::Controller::ControllerBehavior, or do we want those separate?

**Comment by awead** [comment]
@jcoyne looks good with Sufia. Just verify that you want Hydra::Controller::ControllerBehavior separate and I'll merge.

**Comment by jcoyne** [comment]
@awead `Hydra::Controller::ControllerBehavior` is stuff that should be accessible in all application controllers. It's analogous to `Blacklight::Controller`, so I generate it into `ApplicationController` the same way `Blacklight::Controller` is.  `Hydra::Catalog` is stuff that is only relevant to controllers that do searches.

**Pull request merged**